pages/doc/logging_send_logs.md

```diff
@@ -1,24 +1,21 @@
 ---
-title: Send Logs to Tanzu Observability (Beta)
+title: Send Logs to Our Service (Beta)
 keywords: data, logs
 tags: [getting started, logs]
 sidebar: doc_sidebar
 permalink: logging_send_logs.html
-summary: Learn about sending logs to Tanzu Observability.
+summary: Learn about sending logs to our service.
```

Comment:
Learn about sending logs to VMware Aria Operations for Applications (formerly known as Tanzu Observability by Wavefront).